.github/workflows/release.yml

```diff
@@ -5,14 +5,23 @@ on:
         tags:
             - '*'
```

Comment:
This currently runs when you push a tag, but it seems like the action itself pushes tags. 

All of the examples in the actions happen on a push to main, so whenever a PR is merged.

Reply:
Would it be alright to switch it to this?

```yaml
on:
    push:
        branches: [main]
```